pipeline/outputs/kafka.md

```diff
@@ -47,6 +47,45 @@ In your main configuration file append the following _Input_ & _Output_ sections
     Topics      test
 ```
 
+### Windows build guidance
+
+It is assumed that you have installed openssl in C:\Program Files\OpenSSL-Win64
+
+The kafka output plugin will produce an error similar to the following when trying to build for windows
+
+```text
+LINK : warning LNK4044: unrecognized option '/lpthread'; ignored
+LINK : fatal error LNK1104: cannot open file 'libcrypto.lib'
+NMAKE : fatal error U1077: '"C:\Program Files\CMake\bin\cmake.exe"' : return code '0xffffffff'
+```
+
+This is due to the fact that the location of the openssl library required is incorrect. You can fix it by following the steps below
```

Comment:
I think you've missed the actual steps to install/modify OpenSSL installation directory.